modules/local/filter.nf

```diff
@@ -0,0 +1,41 @@
+process FILTER {
```

Comment:
Perhaps some of these local-modules could eventually be moved to the nf-core/modules-repo?

Reply:
I think they are (mostly, if not all) very specific to detaxizer, so i don't think moving them to nf-core/modules would be beneficial.

Reply:
I agree that this would improve the quality of the software, but I think this would be something for the future. Would `nf-test` work for local modules? I think this would be the way to go as the modules are, like I stated above, very specific to detaxizer and I do not think that they could be made more general (in most cases at least).
But I get your point, testing apart from pipeline tests would be beneficial :)

Reply:
I guess `nf-test` should also work for local modules, but I don't know for sure.